CODEOWNERS

```diff
@@ -1,6 +1,8 @@
 # Order is important; the last matching pattern takes the most
 # precedence.
 
+* @tenstorrent/metalium-admins
```

Comment:
We have to review everything that's not caught right now by the current rules?

Do we want that? @afuller-TT @bkeith-TT